rskj-core/src/test/java/co/rsk/peg/BridgeSupportTest.java

```diff
@@ -152,6 +155,93 @@ void voteFeePerKbChange_success() {
         assertEquals(FeePerKbResponseCode.SUCCESSFUL_VOTE.getCode(), result);
     }
 
+    @Test
+    void getLockWhitelistSize() {
+        WhitelistSupport whitelistSupport = mock(WhitelistSupportImpl.class);
+        when(whitelistSupport.getLockWhitelistSize()).thenReturn(Integer.SIZE);
```

Comment:
I am a bit confused, does `getLockWhitelistSize` represent the data size of a particular entry or the max size of the whilelist?

Reply:
Good question! It returns the lock whitelist size, in other words, the number of whitelisted addresses.

Reply:
I see thanks!